### PR TITLE
update nco invocation

### PR DESCRIPTION
**Pull request by chengzhuzhang**
Optimize nco invocation

### CODE REVIEW

File: e3sm_to_cmip/mpas.py [hunk 1]
```diff
@@ -40,8 +40,7 @@ def remap(ds, mappingFileName, threshold=0.05):
     env['NCO_PATH_OVERRIDE'] = 'no'
 
     args = ['ncremap', '-7', '--dfl_lvl=1', '--no_stdin',
-            '--no_cll_msr', '--no_frm_trm', '--no_stg_grd', '--msk_src=none',
-            '--mask_dst=none', '--map={}'.format(mappingFileName), inFileName,
+            'no_cell_measures', 'no_formula_terms', '--map={}'.format(mappingFileName), inFileName,
```
Comment (czender) [inline e3sm_to_cmip/mpas.py:43]: These two options need double-dashes preceding their names, `'no_cell_measures', 'no_formula_terms`. I do not see how a test of this patch could have succeeded.

File: scripts/cwl_workflows/atm-unified-eam/hrzremap_posin.cwl [hunk 2]
```diff
@@ -37,12 +37,10 @@ arguments:
...
   - --dfl_lvl=1
-  - --no_cll_msr
+  - --no_cell_measures
   - --no_stdin
   - -P
-  - "eam"
-  - -a
-  - sdd
+  - eam
   - --regrid_options="--rnr=0.0"
```
Comment (czender) [inline scripts/cwl_workflows/atm-unified-eam/hrzremap_posin.cwl:44]: I suggest changing this option from `--regrid_options="--rnr=0.0"` to `--preserve=local_mean`. It's clearer and it does the same thing in NCO 5.0.2 and later and I believe zppy is currently using NCO 5.0.3.

File: scripts/cwl_workflows/lnd-elm/ncremap_lnd.cwl [hunk 3]
```diff
@@ -14,7 +14,9 @@ arguments:
   - $(inputs.timeout)
   - ncremap
   - -P
-  - "elm"
+  - elm
+  - "-7"
```
Comment (czender) [inline scripts/cwl_workflows/lnd-elm/ncremap_lnd.cwl:18]: Quotes on -7 not necessary

**Review by czender — CHANGES_REQUESTED** [review]
Please address in-line suggestions before merging

**Comment by chengzhuzhang** [comment]
This is resolved with https://github.com/E3SM-Project/e3sm_to_cmip/pull/155

**Pull request closed without merging**